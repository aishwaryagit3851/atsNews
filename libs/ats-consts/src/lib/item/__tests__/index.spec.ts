import {
  TItemTypes,
  GstItemTypes,
  TStockEntryType,
  StockEntryType,
  TDiscountTypes,
  DiscountTypes,
  TVoucherType,
  VoucherTypeObj,
  VoucherType,
  GstItemTypes_OBJ,
  StockEntryType_OBJ,
  DiscountType_OBJ,
  VoucherEntryType
} from '..';

describe('ATS CONSTS for - firebase', () => {
  it('Match consts to snapshot', () => {
    const NewTItemTypes: TItemTypes[] = ['SIMPLE', 'VARIABLE'];
    expect(NewTItemTypes).toMatchSnapshot();

    expect(GstItemTypes).toMatchSnapshot();

    const NewTStockEntryType: TStockEntryType[] = ['IN', 'OUT'];
    expect(NewTStockEntryType).toMatchSnapshot();

    expect(StockEntryType).toMatchSnapshot();

    const NewTDiscountTypes: TDiscountTypes[] = ['FLAT', 'PERCENTAGE'];
    expect(NewTDiscountTypes).toMatchSnapshot();

    expect(DiscountTypes).toMatchSnapshot();

    const NewTVoucherType: TVoucherType[] = [
      'OPENING_STOCK',
      'PURCHASE',
      'PURCHASE_RETURN',
      'SALES',
      'SALES_RETURN',
      'STOCK_TRANSFER'
    ];
    expect(NewTVoucherType).toMatchSnapshot();

    expect(VoucherTypeObj).toMatchSnapshot();
    expect(VoucherType).toMatchSnapshot();

    expect(GstItemTypes_OBJ).toMatchSnapshot();
    expect(StockEntryType_OBJ).toMatchSnapshot();
    expect(DiscountType_OBJ).toMatchSnapshot();
    expect(VoucherEntryType).toMatchSnapshot();
  });
});
