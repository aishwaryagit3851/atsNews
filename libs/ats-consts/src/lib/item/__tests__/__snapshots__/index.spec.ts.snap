// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATS CONSTS for - firebase Match consts to snapshot 1`] = `
[
  "SIMPLE",
  "VARIABLE",
]
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 2`] = `
[
  "SIMPLE",
  "VARIABLE",
]
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 3`] = `
[
  "IN",
  "OUT",
]
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 4`] = `
[
  "IN",
  "OUT",
]
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 5`] = `
[
  "FLAT",
  "PERCENTAGE",
]
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 6`] = `
[
  "PERCENTAGE",
  "FLAT",
]
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 7`] = `
[
  "OPENING_STOCK",
  "PURCHASE",
  "PURCHASE_RETURN",
  "SALES",
  "SALES_RETURN",
  "STOCK_TRANSFER",
]
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 8`] = `
{
  "OPENING_STOCK": "OPENING_STOCK",
  "PURCHASE": "PURCHASE",
  "PURCHASE_RETURN": "PURCHASE_RETURN",
  "SALES": "SALES",
  "SALES_RETURN": "SALES_RETURN",
  "STOCK_TRANSFER": "STOCK_TRANSFER",
}
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 9`] = `
[
  "OPENING_STOCK",
  "SALES",
  "PURCHASE",
  "STOCK_TRANSFER",
  "SALES_RETURN",
  "PURCHASE_RETURN",
]
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 10`] = `
{
  "SIMPLE": "SIMPLE",
  "VARIABLE": "VARIABLE",
}
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 11`] = `
{
  "IN": "IN",
  "OUT": "OUT",
}
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 12`] = `
{
  "FLAT": "FLAT",
  "PERCENTAGE": "PERCENTAGE",
}
`;

exports[`ATS CONSTS for - firebase Match consts to snapshot 13`] = `
{
  "OPENING_STOCK": "IN",
  "PURCHASE": "IN",
  "PURCHASE_RETURN": "OUT",
  "SALES": "OUT",
  "SALES_RETURN": "IN",
  "STOCK_TRANSFER": "OUT",
}
`;
